appstore/appstore/urls.py

```diff
@@ -35,10 +37,17 @@
 
     path('irods/login', app_core_views.IrodsLogin.as_view(), name='irods_login'),
```

Comment:
this can be removed since irods_login no longer exists

Reply:
Can we remove all `irods` code, for instance `python-irodsclient==0.8.2 ` and the IRODS code in `core/views.py`?

Reply:
I don't see why not.

Reply:
I'll include that in a future PR.